scripts/doc/Makefile

```diff
@@ -238,17 +241,15 @@ install:
 	git commit -a -m 'Make install' && \
 	git push
 
+install-pr-user-ssh:
+	rm -rf _build/mne-biomag-group-demo.github.io
+	git clone -b gh-pages --single-branch --no-checkout --depth 1 git@github.com:/$(USER)/mne-biomag-group-demo _build/mne-biomag-group-demo.github.io
+	@make install-files
+
 install-pr-user:
 	rm -rf _build/mne-biomag-group-demo.github.io
-	git clone -b gh-pages --single-branch --no-checkout --depth 1 https://github.com:/$(USER)/mne-biomag-group-demo _build/mne-biomag-group-demo.github.io
-	touch _build/mne-biomag-group-demo.github.io/.nojekyll
-	cd _build/ && \
-	cp -r html/* mne-biomag-group-demo.github.io && \
-	cd mne-biomag-group-demo.github.io && \
-	git add * && \
-	git add .nojekyll && \
-	git commit -a -m 'Make install' && \
-	git push
+	git clone -b gh-pages --single-branch --no-checkout --depth 1 git@github.com:/$(USER)/mne-biomag-group-demo _build/mne-biomag-group-demo.github.io
+	@make install-files
```

Comment:
what is the diff between install-pr-user-ssh and install-pr-user ?

it seems to be the same

Reply:
you're right, I need to revert to using `https://` in this one